### PR TITLE
docs: Fix a couple of broken links in the docs site

### PR DESCRIPTION
**Pull request by julianna-ciq**

### CODE REVIEW

**Review by kriswest — APPROVED** [review]
LGTM

File: website/versioned_docs/version-1.0/appd-spec.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ original_id: appd-spec
 ---
 ## API
 
-View the full specification in [OpenAPI v3.0](https://www.openapis.org/) format (generated with [ReDoc](https://rebilly.github.io/ReDoc/)).
+View the [full specification](/1.0/appd-spec) in [OpenAPI v3.0](https://www.openapis.org/) format (generated with [ReDoc](https://rebilly.github.io/ReDoc/)).
```
Comment (robmoffat) [inline website/versioned_docs/version-1.0/appd-spec.md:8]: /docs/1.0/appd-spec ?

**Review by robmoffat — APPROVED** [review]

**Review by robmoffat — CHANGES_REQUESTED** [review]
think this is a broken link

File: website/versioned_docs/version-1.0/appd-spec.md (hunk 1 above)
Comment (kriswest) [reply website/versioned_docs/version-1.0/appd-spec.md:8]: Nope, that the page on which the link exists, trying to find what the link to the redoc page is... appears to have changed in this PR (both the link on the page and the correct link to the file, chance might need a fix here)

Comment (kriswest) [reply website/versioned_docs/version-1.0/appd-spec.md:8]:
```suggestion
View the [full specification](/schemas/1.0/app-directory) in [OpenAPI v3.0](https://www.openapis.org/) format (generated with [ReDoc](https://rebilly.github.io/ReDoc/)).
```

This'll fix it, looks like you moved the page to match the other versions, so link changed

**Pull request merged**